Source/WebCore/dom/FullscreenManager.cpp

```diff
@@ -308,8 +308,13 @@ void FullscreenManager::cancelFullscreen()
         // This triggers finishExitFullscreen with ExitMode::Resize, which fully exits the document.
         if (auto* fullscreenElement = topDocument->fullscreenManager().fullscreenElement())
             topDocument->page()->chrome().client().exitFullScreenForElement(fullscreenElement);
-        else
+        else {
+#if !RELEASE_LOG_DISABLED
```

Comment:
This isn't the only `INFO_LOG` in this lambda, so it seems like putting
```
#if RELEASE_LOG_DISABLED
             UNUSED_PARAM(this);
#endif
```
at the beginning of the lambda would be more appropriate.

Reply:
Good point. Fixed. Thank you very much.